test-app/app/models/user.rb

```diff
@@ -1,9 +1,16 @@
 class User < ApplicationRecord
   has_one :articles
-  has_many :children, ->(user) { unscope(:where).where("father_id = :id OR mother_id = :id", id: user.id) }, class_name: "User"
+
+  has_many :children,
+  -> (user) { unscope(:where).where("father_id = :id OR mother_id = :id", id: user.id) },
+   class_name: "User"
+
+   has_many :grandchildren,
```

Comment:
Interesting. 
Can you make it into 1 sql query?

Reply:
Can try, abit hard 😂

Reply:
wow. a bit only? ok. ganbatte

Reply:
alot hard 😂

Reply:
still, ganbatte.
:P